config/environments/production.rb

```diff
@@ -43,7 +43,7 @@
   config.session_store :cookie_store, key: "_early_career_framework_session", secure: true, expire_after: 2.weeks
 
   # Use a different cache store in production.
-  # config.cache_store = :mem_cache_store
+  config.cache_store = :redis_cache_store, { url: ENV["REDIS_CACHE_URL"], pool_size: ENV.fetch("RAILS_MAX_THREADS", 5) }
```

Comment:
```suggestion
  config.cache_store = :redis_cache_store, { url: ENV.fetch("REDIS_CACHE_URL"), pool_size: ENV.fetch("RAILS_MAX_THREADS", 5) }
```
just to ensure its not missing :)

Reply:
The issue with doing this is it causes the `assets:precompile` to fail (as the `REDIS_CACHE_URL` isn't set during the docker build, which makes sense). It feels a bit wrong to me to have something like `REDIS_CACHE_URL=fake bundle exec assets:precompile` to work around it 🤔 if the Redis cache is inaccessible Rails will treat it as a cache miss; I've updated with Leandro's suggestion of sending the Redis errors to Sentry as well so we should be notified there if its an issue

Reply:
ok sounds fair 👍